aws/retryer.go

```diff
@@ -152,6 +152,7 @@ func shouldRetryError(origErr error) bool {
 		}
 		// *url.Error only implements Temporary after golang 1.6 but since
 		// url.Error only wraps the error:
+		// Todo: Investigate use case for this error case for +Go1.11
```

Comment:
This todo should have a matching Github issue so we don't loose track of the task.

Reply:
We should have a Github issue that tracks refactoring the SDK's retry behavior to take advantage of the supported Go version's behavior for request failures and cancellation.